src/board.cpp

```
@@ -18,17 +18,12 @@
-    NORMAL,
-    QSEARCH,
-    EVASIONS,
-};
-
-template <PieceColor color, GenerationType type>
-void generateMoves(Bitboard& bitboard, MoveList* moveList);
-} // namespace Zagreus
\ No newline at end of file
+bool Board::isMoveLegal(Move move) {
+    // TODO: Implement
+}
```

cppcheck

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)